### PR TITLE
0.2.0: updated PK

### PR DESCRIPTION
**Pull request by redcatbear**
Updated PK to enable release.

### CODE REVIEW

File: markdown-statistics-tutorial/.settings/org.eclipse.jdt.core.prefs
Comment (kaklakariada) [inline markdown-statistics-tutorial/.settings/org.eclipse.jdt.core.prefs:1]: Mark this as generated

File: pom.xml [hunk 1]
```diff
@@ -84,6 +78,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.1.2</version>
```
Comment (kaklakariada) [inline pom.xml]:
```suggestion
```

Version is defined in generated parent pom

File: tls-tutorial/.settings/org.eclipse.jdt.core.prefs
Comment (kaklakariada) [inline tls-tutorial/.settings/org.eclipse.jdt.core.prefs:1]: Mark file as generated

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (redcatbear) [reply pom.xml]: Done.

File: tls-tutorial/.settings/org.eclipse.jdt.core.prefs
Comment (redcatbear) [reply tls-tutorial/.settings/org.eclipse.jdt.core.prefs:1]: Done.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**